### PR TITLE
Add kagura config

### PR DESCRIPTION
**Pull request by konradybcio**

### CODE REVIEW

**Comment by mariogrip** [comment]
Does this config fully work? if so i can merge it now

**Comment by konradybcio** [comment]
Uh I thought the only way to test it is to merge it :see_no_evil: 

But I literally copied dora which is confirmed working and changed references and the recovery checksum to that they would match kagura.

**Comment by mariogrip** [comment]
> Uh I thought the only way to test it is to merge it see_no_evil
> 
> But I literally copied dora which is confirmed working and changed references and the recovery checksum to that they would match kagura.

you can test using `ubports-install -f [config]` :)

**Comment by konradybcio** [comment]
Give me 5 minutes then :)

**Comment by konradybcio** [comment]
I can't seem to be able to select a release channel, the list is blank.. Is that intended or perhaps it's an error on my side?

**Comment by mariogrip** [comment]
> I can't seem to be able to select a release channel, the list is blank.. Is that intended or perhaps it's an error on my side?

Has the device been added to the system image server?

**Comment by konradybcio** [comment]
It is on CI.. :shrug: 

https://ci.ubports.com/job/Device%20Compatibility%20Images/job/halium-kagura/

**Comment by mariogrip** [comment]
> It is on CI.. shrug
> 
> https://ci.ubports.com/job/Device%20Compatibility%20Images/job/halium-kagura/

its not here yet, https://system-image.ubports.com/16.04/arm64/edge/

Give me a minute, will add it to the system image

**Comment by mariogrip** [comment]
here we go: https://system-image.ubports.com/16.04/arm64/edge/kagura/ 

please try again now :)

**Comment by konradybcio** [comment]
Hmm.. for some reason it keeps rebooting to recovery after the installation is complete.. let me try to investigate..

**Comment by konradybcio** [comment]
Could you trigger a CI rebuild?

**Comment by konradybcio** [comment]
Okay.. so something goes terribly wrong and the rootfs doesn't get installed neither to /data nor to /system.. The files from the installer reside in /data/recovery/ but they don't install..?

**Comment by Flohack74** [comment]
I had similar experience for angler... Currently for some reason the installer and/or system-image code in recovery more or less destroy the installation. It does not seem to detect halium mode correctly...

File: v1/kagura.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      ],
+      "prerequisites": [],
+      "steps": [
+        {
+          "type": "download",
+          "condition": {"var": "bootstrap", "value": true},
+          "group": "firmware",
+          "files": [
+            {
+              "url": "https://ci.ubports.com/job/Device%20Compatibility%20Images/job/halium-kagura/2/artifact/halium-unlocked-recovery_kagura.img",
```
Comment (Flohack74) [inline v1/kagura.json:68]: Make correct version

**Review by Flohack74 — CHANGES_REQUESTED** [review]

File: v1/kagura.json (hunk 1 above)
Comment (konradybcio) [reply v1/kagura.json]: fixed!

**Review by Flohack74 — APPROVED** [review]

**Review by Flohack74 — APPROVED** [review]

**Pull request merged**